packages/client/src/game/reducers/replayReducer.ts

```diff
@@ -252,9 +253,13 @@ function hypoAction(
   // The morph action is handled here. Also take note of any draws that conflict with the known card
   // identities.
   if (action.type === "morph" || action.type === "draw") {
-    if (action.suitIndex === -1 && action.rank === -1) {
+    if (
+      // eslint-disable-next-line isaacscript/strict-enums
+      action.suitIndex === CardIdentityTypes.Original &&
```

Comment:
the type of suitIndex should be `number | CardIdentityType` I think

Reply:
because if you are disabling the rule it probably means you have something incorrectly typed

Reply:
Yeah, i tried that but it didn't work. And i didn't do anything differently in the other place. Tried to find information on Google but didn't find anything, and tried to read the documentation of that rule, but ended up on a 404 page. It works of course of i use "value of" but that's even uglier. I can look a little further

Reply:
i wrote the rule, why is the documentation 404

Reply:
Don't know, might be wrongly linked in my workspace maybe?

Reply:
what link are you clicking on